Src/Utils/children.py

```
@@ -0,0 +1,14 @@
+import dearpygui.dearpygui as dpg
+
+
+def get_children(item: str | int, depth: int = None, slot: int = 1) -> set[int | str]:
```

Ruff (0.15.20)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)